website/docs/cloud-docs/api-docs/policies.mdx

```diff
@@ -40,7 +40,7 @@ description: >-
 
 Policies are rules that Terraform Cloud enforces on Terraform runs. You can use policies to validate that the Terraform plan complies with security rules and best practices. Terraform Cloud policy enforcement lets you use the policy-as-code frameworks Sentinel and Open Policy Agent (OPA) to apply policy checks to Terraform Cloud workspaces. Refer to [Policy Enforcement]((/cloud-docs/policy-enforcement) for more details.
 
-[Policy sets](/terraform/cloud-docs/policy-enforcement/manage-policy-sets) are collections of policies that you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces, in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set and displays the results in the UI.
+[Policy sets](/terraform/cloud-docs/policy-enforcement/manage-policy-sets) are collections of policies that you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects), workspaces and exclusions, in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set and displays the results in the UI.
```

Comment:
I don't think we need this change 😸 Here we are talking about how you can apply policy sets and it's only globally, to workspaces and to projects.

Reply:
fixed!